corehq/apps/sso/tests/base_test.py

```diff
@@ -0,0 +1,24 @@
+from django.test import TestCase
+
+from corehq.apps.sso.tests import generator
+
+
+class BaseIdPTest(TestCase):
```

Comment:
ah I should have pointed this out earlier. We moved away from this sort of "base test" model because it is far less readable for someone who is less familiar with the code. It would be better if this `BaseIdPTest` class was removed and the `setUpClass` methods put back where they were. This pattern was followed intentionally after some back and forth earlier. I wouldn't follow the accounting tests as models for this as that code is much older.

Reply:
No, this was something I added since your last review. I didn't know this was a model we don't prefer. I'll have a look at newer examples and find a better way of doing this. Thanks.